docs/source/getting-started/command-line-interface/run.rst

```diff
@@ -63,3 +63,21 @@ To start the backend server and skip the process cleanup::
 
   # successful output looks the same as without the flag
 
+To enable log parsing for errors, set ``ELASTIC_SEARCH_PATH`` and ``LOGSTASH_PATH`` variables to point to elasticsearch and logstash binaries. For example:
```

Comment:
Can you add a section to the optional dependencies section of the installation docs (`docs/source/getting-started/installation.rst`) about Elasticsearch and Logstash, including links to their respective installation guides? You don't need to go in depth, just the links + maybe a sentence or two explaining what they're used for.